src/utils/styles.js

```diff
@@ -621,9 +622,12 @@ function getFieldStyles(field) {
                 styles.setStyle('placeholderFocus', 'display', 'none');
             }
 
-            styles.apply('tooltip', 'font_size', (a) => ({
+            styles.apply('tooltipIcon', 'font_size', (a) => ({
                 width: `${a}px`
             }));
+            styles.apply('tooltip', 'font_size', (a) => ({
+                fontSize: `${a - 2}px`
```

Comment:
Try replacing this with `fontSize: smaller` and see how that looks

Reply:
https://github.com/feathery-org/feathery-react/pull/171